pubspec.yaml

```diff
@@ -94,17 +94,18 @@ dependencies:
   cupertino_icons: ^0.1.2
 
 dependency_overrides:
-#  flutter_map:
-#    # Use local branch of flutter_map
-##    path: ../flutter_map
-#    git:
-##      url: git://github.com/johnpryan/flutter_map
-#      # Merged PR https://github.com/johnpryan/flutter_map/pull/577 Evict error tiles
-#      # Merged PR https://github.com/johnpryan/flutter_map/pull/584 Observe TileLayerOptions changes
-#      # Proposed fix for missing disposal of AnimationController in Tile class
+  flutter_map:
+    # Use local branch of flutter_map
+#    path: ../flutter_map
+    git:
+      url: git://github.com/johnpryan/flutter_map
```

Comment:
wrong repo, is git://github.com/**fleaflet**/flutter_map

Reply:
Corrected